examples/todo-typescript/src/server/queries.ts

```diff
@@ -0,0 +1,9 @@
+import HttpError from '@wasp/core/HttpError.js';
+import { Context, Task } from './serverTypes'
+
+export const getTasks = async (args: null, context: Context): Promise<Task[]> => {
```

Comment:
Are you sure the type of `args` is `null`? I think it's more likely to be an empty object `{}`.

If you wanted to say "I don't care what type this is" without resorting to `any`, it's best to use `unknown`.